README.md

```diff
@@ -741,6 +756,42 @@ The update command fails if there are changes in your program or library that `m
 
 ### Updating to an upstream version
 
+Before updating a program or a library, it's good to know the names of the stable releases, usually marked with a tag using a common format, such as `1.2`, `v1.0.1`, `r5.6`, `mbed-os-5.6` and so on.
```

Comment:
This line is really helpful for determining what will be considered "stable" release. It looks like you're using a regular expression to determine this.

Would it also be possible to add to the docs the specific rules you're checking for? For instance, "Starts with `v`, `r` `mbed-os-`, etc" and "does not end in `rcxxxx`".